README.md

```diff
@@ -1 +1,62 @@
-#Gregorio AUR repository
+#Gregorio PKGBUILDs for the AUR
+
+This repository provides files for building Gregorio from the [Arch User Repository](https://aur.archlinux.org).
+
+##Packages
+
+Two Gregorio packages are available:
+
+- `gregorio` provides the latest [stable release](https://github.com/gregorio-project/gregorio/releases).
+- `gregorio-git` builds from the `develop` branch of the [Gregorio Github repository](https://github.com/gregorio-project/gregorio/tree/develop).
+
+##Usage
+
+The easiest way to install packages from the AUR is to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers). For example,
+
+    $ yaourt -S gregorio
+
+To install manually from this repository using [`makepkg`](https://wiki.archlinux.org/index.php/Makepkg):
+
+    $ git clone https://github.com/gregorio-project/gregorio-aur.git
+    $ cd gregorio-aur/gregorio
+    $ makepkg -si
+
```

Comment:
makepkg -i runs pacman -U on the generated package.

On Sun, Apr 12, 2015 at 9:37 PM Br. Elijah Schwab, O.Carm <
notifications@github.com> wrote:

> In README.md
> <https://github.com/gregorio-project/gregorio-aur/pull/4#discussion_r28210586>
> :
>
> > +
> > +- `gregorio` provides the latest [stable release](https://github.com/gregorio-project/gregorio/releases).
> > +- `gregorio-git` builds from the `develop` branch of the [Gregorio Github repository](https://github.com/gregorio-project/gregorio/tree/develop).
> > +
> > +##Usage
> > +
> > +The easiest way to install packages from the AUR is to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers). For example,
> > +
> > +    $ yaourt -S gregorio
> > +
> > +To install manually from this repository using [`makepkg`](https://wiki.archlinux.org/index.php/Makepkg):
> > +
> > +    $ git clone https://github.com/gregorio-project/gregorio-aur.git
> > +    $ cd gregorio-aur/gregorio
> > +    $ makepkg -si
> > +
>
> For completeness is it worth adding sudo pacman -U gregorio-???.pkg.tar.xz
> ? Or maybe the link above to makepkg/Arch wiki is good enough.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gregorio-project/gregorio-aur/pull/4/files#r28210586>.
>